src/main/java/org/cactoos/text/BytesAsText.java

```diff
@@ -114,4 +114,9 @@ public String asString() throws IOException {
         return new String(this.bytes.asBytes(), this.charset);
     }
 
+    @Override
+    public int compareTo(final Text text) {
+        return new UncheckedText(text).compareTo(this);
```

Comment:
@alex-semenyuk I agree with you, but the contract wasn't broken. The difference is who will do the compare is `UncheckedText`. Anyway, the alternative do something like:
```java
return new UncheckedText(this).asString().compareTo(new UncheckedText(text).asString());
```
Please, not I can't call `this.asString()` or `text.asString()` inside `compareTo()` because both throw exceptions. So, I need wrap them with `UncheckedText` before call `asString()`. Makes sense?